utils/files.ts

```diff
@@ -0,0 +1,48 @@
+import { RESPONSE_MESSAGES } from "../constants/index";
+import { NextApiResponse } from "next";
+import { Transform } from "stream";
+
+export function valuesToCsvLine(values: string[], delimiter: string): string {
+    return values.join(delimiter);
+}
+interface DataObject {
+    [key: string]: any;
+}
+
+export function getDataFromValues(data: DataObject, headers: string[]): Partial<DataObject> {
+    const result: Partial<DataObject> = {};
```

Comment:
AFAIK here `Partial<DataObject>` is the same as `DataObject`, since there are no explicitly defined keys you are saying could not be present.